pkg/reconciler/v1alpha1/clusteringress/clusteringress.go

```diff
@@ -113,6 +122,14 @@ func NewController(
 		Handler:    reconciler.Handler(impl.EnqueueLabelOfClusterScopedResource(networking.IngressLabelKey)),
 	})
 
+	c.tracker = tracker.New(impl.EnqueueKey, opt.GetTrackerLease())
+	gvk := corev1.SchemeGroupVersion.WithKind("Secret")
+	secretInfomer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    controller.EnsureTypeMeta(c.tracker.OnChanged, gvk),
+		UpdateFunc: controller.PassNew(controller.EnsureTypeMeta(c.tracker.OnChanged, gvk)),
+		DeleteFunc: controller.EnsureTypeMeta(c.tracker.OnChanged, gvk),
```

Comment:
Use `reconciler.Handler` to remove redundancy

Reply:
Done